winml/lib/Telemetry/WinMLTelemetryHelper.cpp

```diff
@@ -106,4 +106,21 @@ void WinMLTelemetryHelper::RegisterOperatorSetSchema(
       TraceLoggingInt32(default_attribute_count, "defaultAttributeCount"),
       TraceLoggingInt32(runtime_session_id_, "runtime_session_id_"),
       TraceLoggingKeyword(MICROSOFT_KEYWORD_MEASURES));
+}
+
+void WinMLTelemetryHelper::SetIntraOpNumThreadsOverride(
+    uint32_t num_threads_override) {
+  if (!telemetry_enabled_)
+    return;
+  WinMLTraceLoggingWrite(
+      provider_,
+      "SetIntraOpNumThreadsOverride",
+      TraceLoggingKeyword(WINML_PROVIDER_KEYWORD_DEFAULT),
+      TelemetryPrivacyDataTag(PDT_ProductAndServiceUsage),
+      //Telemetry info
+      TraceLoggingUInt8(WINML_TLM_NATIVE_API_INTRAOP_THREADS_VERSION, "schemaVersion"),
+      // num threads info
+      TraceLoggingInt32(num_threads_override, "numThreadsOverride"),
+      TraceLoggingInt32(std::thread::hardware_concurrency(), "maxThreadsOnMachine"),
+      TraceLoggingKeyword(MICROSOFT_KEYWORD_MEASURES));
```

Comment:
Nit: can you put this below TraceLoggingKeyword(WINML_PROVIDER_KEYWORD_DEFAULT),

Reply:
Thanks for suggestion but putting it as the last line is the same style as the rest of the trace logging functions.